### ISSUE TITLE
"Unsupported media type" attempting to validate phone numbers with Pinpoint

### ISSUE THREAD

**Issue by cjsauer** [cognitect-labs/aws-api#84]
## Dependencies

``` clojure
com.cognitect.aws/api               {:mvn/version "0.8.305"}
com.cognitect.aws/endpoints         {:mvn/version "1.1.11.568"}
com.cognitect.aws/email             {:mvn/version "722.2.470.0"}
com.cognitect.aws/sns               {:mvn/version "718.2.444.0"}
com.cognitect.aws/pinpoint          {:mvn/version "711.2.411.0"}
```

## Description with failing test case

I'm attempting to use Pinpoint's phone number validation endpoint, and can't seem to get a proper response.

```clojure
(ns myapp.aws.pinpoint
  (:require [cognitect.aws.client.api :as aws]))

(def pinpoint
  (aws/client {:api :pinpoint}))

(aws/validate-requests pinpoint true)

(defn validate-phone-number
  [phone]
  (aws/invoke pinpoint
              {:op :PhoneNumberValidate
               :request {:NumberValidateRequest
                         {:PhoneNumber phone
                          :IsoCountryCode "US"
                          }}}))

(validate-phone-number "+14085550100")
```

Response:

```clojure
{:message "Unsupported Media Type",
 :cognitect.anomalies/category :cognitect.anomalies/incorrect}
```

Looking at the request headers, it doesn't appear that the `Content-Type` header is present, but I'm unsure whether that is truly the root cause:

```
"x-amz-date" = "20190629T172118Z"
"host" = "pinpoint.us-east-1.amazonaws.com"
"authorization" = "AWS4-HMAC-SHA256 Credential=...
```

Amazon's [example request](https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-sms-verify.html) appears to include that header.

**Comment by dchelimsky** [comment]
Fixed in 5e1a88a1f5bba49a2a235336f8f59b81ca89b944.

**Comment by dchelimsky** [comment]
Fixed in 0.8.335.
